src/igvfd/schemas/in_vitro_system.json

```diff
@@ -0,0 +1,199 @@
+{
+    "title": "In Vitro System",
+    "$id": "/profiles/in_vitro_system.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting a sample grown in a in vitro system.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source",
+        "donors",
+        "taxa",
+        "biosample_term",
+        "classification"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "sample.json#/properties"
+        },
+        {
+            "$ref": "biosample.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/url"
+        },
+        {
+            "$ref": "mixins.json#/taxa"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "biosample.json#/dependentSchemas"
+        ],
+        "time_post_factors_introduction": {
+            "required": [
+                "time_post_factors_introduction_units"
+            ],
+            "comment": "Biosample with a specified time_post_factors_introduction requires time_post_factors_introduction_units specification."
+        },
+        "time_post_factors_introduction_units": {
+            "required": [
+                "time_post_factors_introduction"
+            ],
+            "comment": "Biosample with a specified time_post_factors_introduction_units requires time_post_factors_introduction specification."
+        },
```

Comment:
dependentRequired would be shorter and more straight forward

Reply:
done